### DISCUSSION TITLE
文档里好像没有SHA1和SHA256使用说明

### DISCUSSION THREAD

**Discussion by huarzone** [spencerwooo/onedrive-vercel-index#617] [Q&A] [answered]
文档里好像没有SHA1和SHA256使用说明，看有人讨论说的是在.password中添加，如果两个都需要进行添加，格式应该是怎么样子的

**Comment by spencerwooo** [comment]
你指的是？哪里的 SHA1 和 SHA256？`.password` 里面写入的是原始密码，纯文本，不用加密的。

**Reply by huarzone** [reply]
文件下方的

**Comment by huarzone** [comment, marked as answer]
![IMG_20220422_212655](https://user-images.githubusercontent.com/101872541/164723554-c8a44a35-31d7-452c-9392-26fb2c236b6f.jpg)
文件下方的这个sha1和sha256不是用来进行一些大文件信息检验的吗，比如我分享的是一些系统镜像

**Reply by spencerwooo** [reply]
这个是 OneDrive API 提供的，跟本项目没关系，本项目没给提供文件 SHA1 或者 SHA256，你截图显示没有也是因为是 E5 开发者版本的 OneDrive - 国际版 API 返回的内容里是有这两个的。

**Reply by huarzone** [reply]
这个不考虑后期添加文件让他显示这个功能吗？

**Reply by spencerwooo** [reply]
不考虑。已经讨论过：https://github.com/spencerwooo/onedrive-vercel-index/discussions/457
